src/renderer/widgets/WalletDetails/ui/WalletConnectDetails.tsx

```diff
@@ -55,15 +82,53 @@ export const WalletConnectDetails = ({ isOpen, wallet, accounts, onClose }: Prop
     return accountsList;
   }, []);
 
+  const reconnect = () => {
+    connect({
+      chains: getWalletConnectChains(chainsService.getChainsData()),
+      pairing: { topic: accounts[0].signingExtras?.pairingTopic },
+    });
+
+    reconnectStarted();
```

Comment:
Why not call `reconnectStarted` or `reconnected` pass all `connect` arguments there?
In model do reconnect stuff and then connect? 
Right not it's a logic in UI: we init connect (that's fine) but then UI somehow decides to call `reconnectStarted`